src/Components/FullArticle.jsx

```diff
@@ -21,7 +21,15 @@ class FullArticle extends Component {
   componentDidMount() {
     let mainPath = '/api/v1/articles/'
     let articlePath = (this.props.location.state.id)
-    axios.get(mainPath + articlePath).then(response => {
+    let session_headers = {
+      'HTTP_ACCEPT': 'application/json',
+      'access-token': window.localStorage.getItem('access-token'),
+      'token-type': 'Bearer',
+      'client': window.localStorage.getItem('client'),
```

Comment:
do we need this when we turned off the change headers on every request setting in the backend?

Reply:
And if we do, I would like you to use axios interceptors instead

Reply:
interceptor installed and working